addons/map/XEH_postInitClient.sqf

```diff
@@ -47,7 +47,7 @@ GVAR(flashlights) = [] call CBA_fnc_createNamespace;
                 private _unitLight = _player getVariable [QGVAR(flashlight), ["", objNull]];
                 _unitLight params ["_flashlight", "_glow"];
                 if (_mapOn) then {
-                    if (!(_flashlight isEqualTo "") && {isNull _glow}) then {
+                    if (_flashlight isNotEqualTo "") && {isNull _glow}) then {
```

Comment:
Here, too.

Reply:
here the linter is failing because there is a bracket closing to much
```suggestion
                    if (_flashlight isNotEqualTo "" && {isNull _glow}) then {
```